ee/vellum_ee/workflows/display/nodes/vellum/tests/test_prompt_node.py

```diff
@@ -280,3 +281,37 @@ class MyWorkflow(BaseWorkflow):
 
     # AND the legacy source_handle_id should be the default port
     assert my_prompt_node["data"]["source_handle_id"] == "149d97a4-3da3-44a9-95f7-ea7b8d38b877"
+
+
+def test_serialize_node__prompt_parameters__dynamic_references():
+    # GIVEN input definition
+    class MyInputs(BaseInputs):
+        input_value: str
+
+    # AND a prompt node with PromptParameters containing dynamic references
+    class DynamicPromptNode(InlinePromptNode):
+        blocks = []
+        ml_model = "gpt-4o"
+        parameters = PromptParameters(custom_parameters={"json_schema": MyInputs.input_value})
+
+    # AND a workflow with the prompt node
+    class Workflow(BaseWorkflow[MyInputs, BaseState]):
+        graph = DynamicPromptNode
+
+    # WHEN the workflow is serialized
+    workflow_display = get_workflow_display(workflow_class=Workflow)
+    serialized_workflow: dict = workflow_display.serialize()
+
+    # THEN the node should properly serialize the PromptParameters
+    dynamic_prompt_node = next(
+        node
+        for node in serialized_workflow["workflow_raw_data"]["nodes"]
+        if node["id"] == str(DynamicPromptNode.__id__)
+    )
+
+    # AND the parameters should be properly serialized in exec_config
+    exec_config = dynamic_prompt_node["data"]["exec_config"]
+    assert "parameters" in exec_config
+
+    parameters = exec_config["parameters"]
+    assert parameters == {}
```

Comment:
Add more assertions at the bottom of this test that assert the shape of the `parameters` attribute